File: 01_Language/01_Functions/php/ctype_print.php
<?php

$strings = array('string1' => "asdf\n\r\t", 'string2' => 'arf12', 'string3' => 'LKA#@%.54', "string4" => "\t");
foreach ($strings as $name => $testcase) {
    if (ctype_print($testcase)) {
        echo "The string '$name' consists of all printable characters.\n";
    } else {
        echo "The string '$name' does not consist of all printable characters.\n";
    }
}

File: 01_Language/01_Functions/python/ctype_print.py
# coding: utf-8

strings = {'string1': "asdf\n\r\t", 'string2': 'arf12', 'string3': 'LKA#@%.54', "string4": "\t"}
for k, v in strings.items():
    if v.isprintable():
        print("The string " + k + " consists of all printable characters.")
    else:
        print("The string " + k + " does not consist of all printable characters.")
